packages/@aws-cdk/aws-certificatemanager/lambda-packages/dns_validated_certificate_handler/package.json

```diff
@@ -33,6 +33,7 @@
     "eslint-plugin-node": "^8.0.1",
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-standard": "^4.0.0",
+    "jest": "^24.1.0",
```

Comment:
I guess we can, but I explicitly added jest to the top-level devDependencies as an encouragement to start using it more in the rest of the codebase. So I'd rather you get rid of it here.

Reply:
Without it being listed here, `npm run test` in the Lambda's directory will not work.

There's no reason we can't have it in both places, though.